reference/customers.v3.yml

```diff
@@ -5083,3 +5140,26 @@ components:
           description: |-
             * Determines if a channel allows global customer to login
             * Determines if customers created on this channel will get global access/login
+    ValidateCustomerCredentialsObject:
+      title: ValidateCustomerCredentialsObject
+      type: object
+      required:
+        - email
+        - password
+      properties:
+        email:
+          type: string
+          description: Customer email address
+        password:
+          type: string
+          description: Customer password
+        channel_id:
+          type: integer
+          description: ChannelId to check the customer credentials against - If this field is empty we will use channel 1
+    ValidateCustomerCredentialsResponseObject:
+      title: ValidateCustomerCredentialsResponseObject
+      type: object
+      properties:
+        is_valid:
```

Comment:
Missing `customer_id`, which get returned if credentials are valid? e.g.:
```
customer_id:
  type: integer
  nullable: true
  example: 1
```

Reply:
🤦  I looked at the wrong screen where it wasn't returned. yeah will update